### PR TITLE
Use isodate to parse version

### PR DESCRIPTION
**Pull request by mrparalon**
I did some profiling using yappi. Header parsing is the second cpu consuming thing after json serialization

```
Sorted by tsub - How much time this function has spent in total, subcalls excluded.
name                                  ncall  tsub      ttot      tavg                                                                                                               
..i/encoders.py:101 jsonable_encoder  445/5  0.004316  0.010260  0.000023                                                                                                           
..hon3.11/_strptime.py:309 _strptime  100    0.002192  0.005411  0.000054                                                                                                           
..1/asyncio/tasks.py:35 current_task  418    0.001467  0.002333  0.000006
```

`fromisoformat` load cpu much less

### CODE REVIEW

**Review by zmievsa — CHANGES_REQUESTED** [review]
Big thanks! A nice and small optimization. Feels safe too. Please, update the version (the minor one) and added a changelog entry.

**Comment by mrparalon** [comment]
@zmievsa thanks, fixed

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 All notable changes to this project will be documented in this file.
 Please follow [the Keep a Changelog standard](https://keepachangelog.com/en/1.0.0/).
 
+## [0.1.2] - 2024-01-15
```
Comment (zmievsa) [inline CHANGELOG.md:6]: @mrparalon This is not a minor version, this is a bugfix version. Please, use the minor version
```suggestion
## [0.2.0] - 2024-01-15
```

File: pyproject.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "verselect"
-version = "0.1.1"
+version = "0.1.2"
```
Comment (zmievsa) [inline pyproject.toml:3]:
```suggestion
version = "0.2.0"
```

**Review by zmievsa — CHANGES_REQUESTED** [review]

**Review by zmievsa — APPROVED** [review]

**Pull request merged**